sympy/ntheory/factor_.py

```diff
@@ -1479,3 +1478,37 @@ def eval(cls, n, k=1):
             else:
                 return Mul(*[(p**(k*(e + 1)) - 1)/(p**k - 1) if k != 0
                            else e + 1 for p, e in factorint(n).items()])
+
+
+def digitslist(n, b=10):
+    """
+    Return a list of the digits of n in base b.
+
+    digitslist(n, 2) is equivalent to [int(d) for d in format(n,'b')]
```

Comment:
This was requested in http://osdir.com/ml/sympy/2013-11/msg00042.html and the digits function in gmpy only works for bases between 2 and 62.